lib/basedriver/driver.js

```diff
@@ -243,6 +243,35 @@ class BaseDriver extends Protocol {
       BaseDriver.DRIVER_PROTOCOL.MJSONWP;
   }
 
+  /**
+   * Check whether a given feature is enabled via its name
+   *
+   * @param {string} name - name of feature/command
+   *
+   * @returns {Boolean}
+   */
+  isFeatureEnabled (name) {
+    // if we specifically have allowed the feature, return true
+    if (this.allowInsecure && this.allowInsecure.includes(name)) {
```

Comment:
you can use _.include, which is more flexible while handling null/undefined args